src/lib/containers/UpsetPlot.tsx

```diff
@@ -142,6 +151,15 @@ const UpsetPlot: React.FunctionComponent<UpsetPlotProps> = ({
           )}
         </SizeMe>
       }
+      {error && 
```

Comment:
you can use our [Error](https://github.com/Sage-Bionetworks/Synapse-React-Client/blob/ed2978c0e238c8e6d17e203bf3dc39ad66813b8e/src/lib/containers/Error.tsx) component instead of the boostrap alert!

Reply:
Saw that, but it looks like that one is currently specific to SynapseClientError.  If it was modified to be more generic (in the current case I run into a TypeError), then that's a great option!

Reply:
ah right, in the future I can add an `errorString` prop and `backendError` or something like that to distinguish the two

Reply:
Alternatively, you could pass in a single error object, and the component examines it to figure out what information it has (then in my case I could pass in the err, and it might be a SynapseClientError or a TypeError!).